janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLStoreManager.java

```diff
@@ -248,7 +256,25 @@ Cluster initializeCluster() throws PermanentBackendException {
             }
         }
 
-        return builder.build();
+        // Build the PoolingOptions based on the configurations
+        PoolingOptions poolingOptions = new PoolingOptions();
+        poolingOptions
+            .setMaxRequestsPerConnection(
+                    HostDistance.LOCAL,
+                    configuration.get(LOCAL_MAX_REQUESTS_PER_CONNECTION))
+            .setMaxRequestsPerConnection(
+                    HostDistance.REMOTE, 
+                    configuration.get(REMOTE_MAX_REQUESTS_PER_CONNECTION));
+        poolingOptions
+            .setConnectionsPerHost(
+                    HostDistance.LOCAL,
+                    configuration.get(LOCAL_CORE_CONNECTIONS_PER_HOST),
+                    configuration.get(REMOTE_CORE_CONNECTIONS_PER_HOST))
+            .setConnectionsPerHost(
+                    HostDistance.REMOTE,
+                    configuration.get(LOCAL_MAX_CONNECTIONS_PER_HOST),
```

Comment:
I think `REMOTE_CORE_CONNECTIONS_PER_HOST` and `LOCAL_MAX_CONNECTIONS_PER_HOST` are switched.

Reply:
You're right. fixed. Thanks

Reply:
@robertdale Please review again.